### PR TITLE
[DX-731]add see also layout -For related content

### PR DESCRIPTION
**Pull request by yurisasuke**
### For internal users - please add a Jira DX PR ticket to the subject!
---
DX-731
### Preview Link
https://deploy-preview-3320--tyk-docs.netlify.app/docs/nightly/tyk-cloud/what-is-tyk-cloud/
https://deploy-preview-3320--tyk-docs.netlify.app/docs/nightly/frequently-asked-questions/faq/
https://deploy-preview-3320--tyk-docs.netlify.app/docs/nightly/getting-started/
https://deploy-preview-3320--tyk-docs.netlify.app/docs/nightly/getting-started/quick-start/tyk-demo/

### Description
- This will show content related to a page using hugo related content feature.
https://gohugo.io/content-management/related/#configure-related-content

### CODE REVIEW

File: tyk-docs/assets/scss/_structure.scss [hunk 1]
```diff
@@ -140,6 +140,25 @@ body {
     font-family: "Inter", sans-serif;
     font-weight: bold;
 }
+
+
+.see_also_header{
```
Comment (letzya) [inline tyk-docs/assets/scss/_structure.scss]: can we call this `see_also_heading`

Comment (yurisasuke) [reply tyk-docs/assets/scss/_structure.scss]: > can we call this `see_also_heading`

yes that a better name.

**Comment by letzya** [comment]
@Keithwachira maybe we should remove it from the overview pages since they already have links? https://deploy-preview-3320--tyk-docs.netlify.app/docs/nightly/getting-started/

**Review by letzya — APPROVED** [review]

**Pull request merged**